### DISCUSSION TITLE
Support for authorization code flow | OIDC4VCI

### DISCUSSION THREAD

**Discussion by vanhoanHoang** [walt-id/.github#47] [Help] [answered]
Dear team, 

I have been working on WaltID products for some time. Actually the ISSUER implements the OIDC4VCI protocols. The later has 2 different flows: Authorization Code and Pre Authorized Code. 

As I can see on the ISSUER portal, I clearly see that Walt supports the Pre-authorized code flow with user pin.  Does WaltID also support the Authorization code please ? If so, how can we use it ?

Thank you very much

**Comment by taminobaumann** [comment, marked as answer]
Hey @vanhoanHoang, 

thanks for reaching out! Yes, we do support the Authorization Code Flow.

You can find an example here: 
https://docs.walt.id/v/ssikit/concepts/oidc/credential-issuance/usage-examples

Best
Tamino
